cmd/admin-handlers.go

```diff
@@ -2778,6 +2834,22 @@ func (a adminAPIHandlers) InspectDataHandler(w http.ResponseWriter, r *http.Requ
 	}); err != nil {
 		logger.LogIf(ctx, err)
 	}
+
+	ci, err := getClusterInfoWithTimeout(ctx, objLayer, 10*time.Second)
+	if err != nil {
+		logger.LogIf(ctx, err)
```

Comment:
Just log and ignore the failure.

Reply:
> Just log and ignore the failure.

Why ignoring the failure ?

Reply:
Because it is not important that we get this. If is isn't added there is no reason to drop the entire request.